apps/language_server/test/providers/execute_command/manipulate_pipes/ast_test.exs

```diff
@@ -0,0 +1,159 @@
+defmodule ElixirLS.LanguageServer.Providers.ExecuteCommand.ManipulatePipes.ASTTest do
+  use ExUnit.Case
+
+  alias ElixirLS.LanguageServer.Providers.ExecuteCommand.ManipulatePipes.AST
+
+  describe "to_pipe/1" do
+    test "single-line selection with two args in named function" do
```

Comment:
Add some tests on negative examples:
- to_pipe on 0 arity function call
- to_pipe on atom
- to_pipe on calls with no parens, e.g. `X.fun`, `Y.fun "abc"`
- already a piped call, e.g. `x |> Y.fun`

Reply:
The current code doesn't pipe this correctly. It pipes `a + b` to `a |> +b`, which raises `(ArgumentError) piping into a unary operator is not supported`

Maybe we could add infix operators to a blocklist of pipeable functions, like you did with `:__aliases__` and so on. How does that sound?

Reply:
> Maybe we could add infix operators to a blocklist of pipeable functions, like you did with :__aliases__ and so on. How does that sound?

If that's easier to do then I'm ok with that. It's better to fail fast than break user's code.

Reply:
Treating this correctly would perhaps be impossible. For piping you need the fully qualified name, but due to the way we parse things, we can't actually resolve the module name, given that infix operators are always imported from somewhere.

I'll add a check for operators to be skipped